test/components/renku_data_services/notebooks/test_notebooks_image_checks.py

```diff
@@ -199,7 +199,8 @@ async def test_public_image_check(image: str, exists_expected: bool) -> None:
     "image,expected_path",
     [
         ("jupyter/minimal-notebook:x86_64-python-3.11.6", PurePosixPath("/home/jovyan")),
-        ("nginx", None),
+        ("nginx", PurePosixPath("/")),
```

Comment:
Note that this test was previously wrong: the image working dir for `nginx` is `/`, not `None`.